cypress/integration/blocks.js

```diff
@@ -596,5 +596,35 @@ if (Cypress.env('API') !== 'guillotina') {
     //   //   cy.contains(expected);
     //   // }
     // });
+
+    it.only('Add Table of Contents block', () => {
+      const h2 = 'expectedH2';
+
+      //Add H2
+      cy.get('.block.inner.text .public-DraftEditor-content')
+        .type(` ${h2}`)
+        .setSelection(` ${h2}`);
```

Comment:
@steffenri why is .setSelection necessary here?

Reply:
using .setSelection here allows us to style in draftJS. In this case we can make the selected text a h2.